.github/workflows/editorconfig.yml

```
@@ -1,7 +1,7 @@
 name: "Checking EditorConfig"
 
```

This will need restrictions on what it can do, for example:

```suggestion
permissions: read-all
```